src/appkit.cs

```diff
@@ -2026,7 +2071,7 @@ interface NSButtonCell {
 	}
 	
 	[BaseType (typeof (NSControl))]
-	interface NSButton : NSAccessibilityButton {
+	interface NSButton : NSAccessibilityButton, NSUserInterfaceCompression {
```

Comment:
@timrisi no, this means that the native type *must* support it so we can generate the methods that calls it